3-platform/warehouse/db_postgres/src/main/resources/db/changelog/changes/02-create-products-table.sql

```diff
@@ -0,0 +1,21 @@
+-- products table
+
+CREATE TABLE public.products (
+                                 id varchar NOT NULL,
+                                 "name" varchar NOT NULL,
+                                 description varchar NULL,
+                                 image varchar NULL,
+                                 price bigint NOT NULL,
+                                 category varchar NULL,
+                                 CONSTRAINT products_pk PRIMARY KEY (id),
+                                 CONSTRAINT products_price_check CHECK (price>=0)
+);
+
+-- Column comments
+
+COMMENT ON COLUMN public.products.id IS 'PK of product record';
```

Comment:
pls follow up that other team members align with your style of sql.

Reply:
e.g. using Comment ON COLLUMN

Reply:
It's possible to create a table using IDE and after that generate DDL script. And "COMMENT" - it's IDE's creature.
So, let's don't make our colleagues change uppercase.